src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionArgumentTests.cs

```diff
@@ -1414,7 +1414,9 @@ static void Main()
                 }
                 """;
             var comp = CreateCompilation([sourceA, sourceB], targetFramework: TargetFramework.Net80);
-            // PROTOTYPE: [with(x)] and [with(x), y] should result in errors since x should not be included in the params argument.
+            // https://github.com/dotnet/roslyn/issues/77866: [with(x)] and [with(x), y] should
+            // result in errors since x should not be included in the params argument. Should
+            // be fixed when the last parameter of the builder method is the items parameter.
```

Comment:
Is this saying we're going to add some enforcement later which will avoid including the 'with' arguments for the last parameter of Create? It might be good to demonstrate what arguments are actually being passed for `items` in `Create(items)`, e.g. with `expectedOutput`. But if it's thought to be too much of a corner case, we can just wait to take the fix instead.

Reply:
Yes, I expect the change for parameter order will include some enforcement to catch cases where `with` arguments may be used for the last parameter.